inflect/__init__.py

```diff
@@ -3001,29 +3001,41 @@ def _handle_prepositional_phrase(cls, phrase, transform, sep):
         return " ".join(
             parts[: pivot - 1] + [sep.join([transformed, parts[pivot], ''])]
         ) + " ".join(parts[(pivot + 1) :])
-
+    
     def _handle_long_compounds(self, word: Words, count: int) -> Union[str, None]:
         """
         Handles the plural and singular for compound `Words` that
         have three or more words, based on the given count.
-
-        >>> engine()._handle_long_compounds(Words("pair of scissors"), 2)
-        'pairs of scissors'
-        >>> engine()._handle_long_compounds(Words("men beyond hills"), 1)
-        'man beyond hills'
```

Comment:
The removal of these tests is concerning. These are here not only to document the expected behavior of this method, but also to serve to protect those behaviors. We should not remove these expectations without a good justification.